### ISSUE TITLE
WhatsApp and Zoom

### ISSUE THREAD

**Issue by Dudematters** [meetfranz/franz#1685]
Every day and multiple times a day Whatsapp is giving me issues, see screenshots. Even when Franz is open and running (just by switching between messengers in Franz). I get the "Update Google Chrome" and the "Update available, click to update WhatsApp >" screens. My Chrome is the latest version and so is WhatsApp. I've cleared the cache, within the app and withing Finder but no change. I took of the 'beta versions', downloaded the stable version, removed all Franz related files and folders from my mac (Mojave 10.14.6) restarted, installed the clean version. Problem stays. Very annoying. Now having done that, the zoom function does not work anymore too "Command +" and "Command -" with keyboard or even selecting from the menu.

I expect WhatsApp to be running normally, not have to refresh my screen "Command R" every time and update whatsapp every time I need to use it.

**Screenshots**
<img width="1060" alt="ScreenShot 2019-11-08 at 11 38 42" src="https://user-images.githubusercontent.com/40238967/68471105-c6276880-021d-11ea-9fc3-cdd04d00c094.png">
<img width="606" alt="ScreenShot 2019-11-08 at 11 39 08" src="https://user-images.githubusercontent.com/40238967/68471118-cb84b300-021d-11ea-8ab7-e0559600d9cf.png">

**Desktop (please complete the following information):**
 - OS: MacOS Mojave 10.14.6
 - Chrome Version 78.0.3904.97
 - Franz Version 5.4.1

**Smartphone (please complete the following information):**
 - Device: Oneplus 6
 - OS: Android 9
 - Chrome 
 - Version 78.0.3904.96

**Additional context**
This problem has been going on for a while now, googling a couple of months ago I read much more people had this problem. Then it was solved for a while, but now it has come back, and more persistent as it happens now even when Franz remains running between switching messengers (in Franz)

**Comment by bittner** [comment]
It's usually a good idea to open separate issues for distinct bugs.

* The zooming issue is fixed with PR #1668, which is waiting to be merged.
* I can confirm the WhatApp + Chrome issue has returned. Not sure though if this is the Chrome/Chromium's project fault or Franz's. For the time being at least it's just disturbing and resolved by reloading.

**Comment by Dudematters** [comment]
Yes, sorry, I know it is better to split issues. Will do next time.

I've investigated a bit, and to be honest I've tested it on other messenger apps too on the Mac. Most of them had the same problem as Franz has (Rambox, Manageyum, etc) but one does not, and works perfectly: All-in-One Messenger (on macOS, on iPad Pro and some bit older ChromeOS it was near to unusable as it lags a lot, where Franz for Linux on ChromeOS was much more useable).
 
So I cannot tell if it's a Franz issue or Chrome issue, as All-in-One Messenger does the job very smooth and perfect, not one time it went wrong on Whatsapp till now. And the zooming in All-in-One Messenger also works very reliably.
